aea/cli_gui/__init__.py

```diff
@@ -76,7 +77,7 @@ def read_description(dir_name, yaml_name):
             if "description" in yaml_data:
                 return yaml_data["description"]
         except yaml.YAMLError as exc:
-            print(exc)
+            logging.error(exc)
```

Comment:
Instead of using the `logging` module directly, I'd rather declare a `logger = logging.getLogger(__name__)` at the beginning of the file and then use `logger.fun` whenever needed.